controllers/clustergroupupgrade_controller.go

```diff
@@ -1501,9 +1559,19 @@ func (r *ClusterGroupUpgradeReconciler) buildRemediationPlan(
 	return nil
 }
 
-func getResourceName(clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, initialString string) string {
-	return clusterGroupUpgrade.Name + "-" + initialString
-}
+// func getGeneratedResourceNamePrefix(clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, initialString string) string {
```

Comment:
Hmm, do we need to keep this code since it's commented out?

Reply:
Meant to remove this too